spec/helper.rb

```diff
@@ -59,4 +59,8 @@ def process(body, content_type = nil, options = {})
   config.expect_with :rspec do |c|
     c.syntax = :expect
   end
+
+  def jruby?
+    defined?(RUBY_ENGINE) and 'jruby' == RUBY_ENGINE
```

Comment:
Style note: prefer `&&` over `and` unless you're using the special precedence of `and`.